### PR TITLE
Synchronize eye joint and debugging vectors

### PR DESCRIPTION
**Pull request by howard-stearns**
when the other avatar is  looking at me:
1) Update eye rotations after the other joints have been set
2) cut down the time on the latch for the islookingatme test.

### CODE REVIEW

File: interface/src/avatar/Head.cpp [hunk 1]
```diff
@@ -364,7 +378,7 @@ void Head::setCorrectedLookAtPosition(glm::vec3 correctedLookAtPosition) {
 bool Head::isLookingAtMe() {
     // Allow for outages such as may be encountered during avatar movement
     quint64 now = usecTimestampNow();
-    const quint64 LOOKING_AT_ME_GAP_ALLOWED = 1000000;  // microseconds
+    const quint64 LOOKING_AT_ME_GAP_ALLOWED = (5 * 1000.0f * 1000.0f) / 60.0f; // n frames, in microseconds
```
Comment (ZappoMan) [inline interface/src/avatar/Head.cpp]: why are these floats? why aren't they all floats and/or quint64s?

File: interface/src/avatar/SkeletonModel.cpp [hunk 2]
```diff
@@ -105,26 +105,37 @@ void SkeletonModel::updateRig(float deltaTime, glm::mat4 parentTransform) {
...
+        params.worldHeadOrientation = head->getFinalOrientationInWorldFrame();
+        params.eyeLookAt = head->getLookAtPosition();
+        params.eyeSaccade = head->getSaccade();
         params.leanJointIndex = geometry.leanJointIndex;
         params.neckJointIndex = geometry.neckJointIndex;
         params.leftEyeJointIndex = geometry.leftEyeJointIndex;
         params.rightEyeJointIndex = geometry.rightEyeJointIndex;
 
         _rig->updateFromHeadParameters(params);
+    } else if (true) { //(_owningAvatar->getHead()->isLookingAtMe()) {
```
Comment (ZappoMan) [inline interface/src/avatar/SkeletonModel.cpp]: dead code?

File: interface/src/avatar/Head.cpp (hunk 1 above)
Comment (howard-stearns) [inline interface/src/avatar/Head.cpp]: good catch. thanks. (Was racing to get it in before the network went down.)

File: interface/src/avatar/SkeletonModel.cpp (hunk 2 above)
Comment (howard-stearns) [inline interface/src/avatar/SkeletonModel.cpp]: Nope. I need to fix that! Rushing...:-(

**Comment by hifi-gustavo** [comment]
Build console output for commit: eeeaa8050d75f6fd796832e25b3150ddb435da2d available at: https://gist.github.com/c41d38a4587ac80eb019<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5573.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5573.dmg

**Comment by ZappoMan** [comment]
Do you still want this tagged as "DO NOT MERGE" or is it ready to go?

**Comment by howard-stearns** [comment]
@ZappoMan: I have more testing to do that I cannot do from home. (Unfortunate timing of the electrician's visit to the office on Friday.)

**Comment by hifi-gustavo** [comment]
Build console output for commit: 941b966dc745fa1c0175feb2a25c4ab4387aa18b available at: https://gist.github.com/c41d38a4587ac80eb019<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5573.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5573.dmg

**Pull request merged**